services/api/src/models/definitions/user.json

```diff
@@ -113,6 +114,11 @@
         "otp"
       ]
     },
+    "smsSecret": {
+      "type": "String",
+      "readScopes": "none",
+      "writeScopes": "none"
```

Comment:
This should change to `read/writeAccess` with the new updates

Reply:
thanks let me confirm that works correctly